net/modemmanager/Makefile

```diff
@@ -0,0 +1,137 @@
+#
+# Copyright (C) 2016 Velocloud Inc.
+# Copyright (C) 2016 Aleksander Morgado <aleksander@aleksander.es>
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=modemmanager
+PKG_VERSION:=1.10.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=ModemManager-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://www.freedesktop.org/software/ModemManager
+PKG_HASH=fa556bb4ba1f1478e709b71151b4de8f7d136d80e51c64e887fc1fd8926cdc70
+PKG_BUILD_DIR:=$(BUILD_DIR)/ModemManager-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Nicholas Smith <nicholas.smith@telcoantennas.com.au>
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+
+define Package/modemmanager/config
+  source "$(SOURCE)/Config.in"
+endef
+
+define Package/modemmanager
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Control utility for any kind of mobile broadband modem
+  URL:=https://www.freedesktop.org/wiki/Software/ModemManager
+  LICENSE:=GPL-2.0-or-later
+  LICENSE_FILES:=COPYING COPYING.LIB
+  DEPENDS:= \
+	$(INTL_DEPENDS) \
+	+glib2 \
+	+dbus \
+	+MODEMMANAGER_WITH_MBIM:libmbim \
+	+MODEMMANAGER_WITH_QMI:libqmi
+endef
+
+define Package/modemmanager/description
+  ModemManager is a D-Bus-activated service which allows controlling mobile
+  broadband modems. Add kernel modules for your modems as needed.
+  Select Utilities/usb-modeswitch if needed.
+endef
+
+CONFIGURE_ARGS += \
```

Comment:
Maybe it make sense to disable documentation here as well?

Reply:
if I'm not mistaken, gtk-doc based documentation will anyway not be built because the build system detects that there isn't any gtk-doc support. But adding --disable-gtk-doc explicitly wouldn't harm anyway.